### ISSUE TITLE
Optionally add the ability to slightly enhance the European archer. 

### ISSUE THREAD

**Issue by Russianstory** [UnofficialCrusaderPatch/UnofficialCrusaderPatch#916]
Well this is just my dream. Just use a separate button to add an arrow boost to a European archer. Let's try to play like that? What if it works out? Let's do an experiment?

**Comment by Altaruss28** [comment]
Like a one-option rebalance in the same manner as for the units the tab already features?

**Comment by Russianstory** [comment]
Yes, exactly as on a separate tab with units - where were the spearmen, Arab swordsmen, ladders?
Let's say someone wants to strengthen the archers a little and he chooses this option. I understand that this will affect the balance. Therefore, I ask you to EXPERIMENTALLY try to do it as a separate option? It seems to me that it will be interesting. Just at least try to play with the reinforced euro archers.

**Comment by Altaruss28** [comment]
Sooner or later, a whole section in which users can make their custom set of rebalances for all units/economy will be added, thus no need for this.
